The -XCClinker flag is libtool-specific; gcc itself does not understand it.
Since EXTRA_LDFLAGS is added to sdl2.pc and sdl2-config, leaving it in breaks
linking directly with $CC (e.g. in SDL2_* configure tests).

--- origsrc/SDL2-2.0.1/build-scripts/ltmain.sh	2013-10-23 23:05:29.000000000 -0500
+++ src/SDL2-2.0.1/build-scripts/ltmain.sh	2013-11-24 20:48:09.837334400 -0600
@@ -5853,7 +5853,7 @@ func_mode_link ()
       # -O*, -flto*, -fwhopr*, -fuse-linker-plugin GCC link-time optimization
       -64|-mips[0-9]|-r[0-9][0-9]*|-xarch=*|-xtarget=*|+DA*|+DD*|-q*|-m*| \
       -t[45]*|-txscale*|-p|-pg|--coverage|-fprofile-*|-F*|@*|-tp=*|--sysroot=*| \
-      -O*|-flto*|-fwhopr*|-fuse-linker-plugin)
+      -O*|-flto*|-fwhopr*|-fuse-linker-plugin|-static-libgcc)
         func_quote_for_eval "$arg"
 	arg="$func_quote_for_eval_result"
         func_append compile_command " $arg"
--- origsrc/SDL2-2.0.1/configure.in	2013-10-23 23:05:29.000000000 -0500
+++ src/SDL2-2.0.1/configure.in	2013-11-24 20:23:58.544325200 -0600
@@ -2597,7 +2597,7 @@ AC_HELP_STRING([--enable-render-d3d], [e
             have_loadso=yes
         fi
         # Set up the system libraries we need
-        EXTRA_LDFLAGS="$EXTRA_LDFLAGS -luser32 -lgdi32 -lwinmm -limm32 -lole32 -loleaut32 -lshell32 -lversion -luuid -XCClinker -static-libgcc"
+        EXTRA_LDFLAGS="$EXTRA_LDFLAGS -luser32 -lgdi32 -lwinmm -limm32 -lole32 -loleaut32 -lshell32 -lversion -luuid -static-libgcc"
         # The Windows platform requires special setup
         VERSION_SOURCES="$srcdir/src/main/windows/*.rc"
         SDLMAIN_SOURCES="$srcdir/src/main/windows/*.c"
